types/serverless/plugins/aws/provider/awsProvider.d.ts

```diff
@@ -498,6 +498,11 @@ declare namespace Aws {
         cloudFront?: CloudFront;
     }
 
+    interface FileSystemConfig {
+        arn: string;
```

Comment:
arn can be string of CF GetAtt, Join and ImportValue, see https://github.com/serverless/serverless/blob/master/lib/plugins/aws/provider/awsProvider.js#L792-L811

Can you put `any` in the mean time if those type are not yet available ?
Or create those as well :) ?